.travis.yml

```diff
@@ -16,6 +16,7 @@ before_install:
     - travis_retry conda create --yes -n test python=$PYTHON --file requirements.txt
     - source activate test
     - conda install --yes --file requirements-dev.txt;
+    - pip install git+https://github.com/python-visualization/branca.git
```

Comment:
> Depending on your reply, I'll either freeze a branca commit into this line, or create a v0.0.0 of branca and ask for it in this line.

Actually that depends on your plans.  Even though I recommend a release (`v0.1.0` at least), we can deal with Travis-CI to pointing `master` until the dust settles.  I just don't want that to become a permanent thing.

So,

- if you see a fast development pace in the next few weeks leave it as-is and let's open an issue to remind us to change it later;
- if you are not going to be developing `branca` for a while and/or if you thing other might be interested into using this ASAP, then let's tag a release.